### PR TITLE
sigh: fix missing name display in build and webpack steps

### PR DESCRIPTION
**Pull request by mykmartin**

### CODE REVIEW

File: tools/sigh.ts [hunk 1]
```diff
@@ -363,6 +363,9 @@ function buildPath(path: string, preprocess: () => void, deps: string[]): () =>
 
     return true;
   };
+  // Using a lambda breaks the display of func.name in the main execution loop.
+  Object.defineProperty(fn, 'name', {value: 'build'});
```
Comment (Cypher1) [inline tools/sigh.ts]: Seem like an opportunity to show the path that is being built with ``` `building ${path}` ```?

File: tools/sigh.ts [hunk 2]
```diff
@@ -472,13 +475,16 @@ function licenses(): boolean {
...
-  return () => {
+  const fn = () => {
     const result = saneSpawnWithOutput('npm', ['run', `build:${pkg}`]);
     if (result.stdout) {
       console.log(result.stdout);
     }
     return result.success;
   };
+  // Using a lambda breaks the display of func.name in the main execution loop.
+  Object.defineProperty(fn, 'name', {value: 'webpack'});
```
Comment (Cypher1) [inline tools/sigh.ts]: Similar here.

File: tools/sigh.ts (hunk 1 above)
Comment (mykmartin) [reply tools/sigh.ts]: Done.

File: tools/sigh.ts (hunk 2 above)
Comment (mykmartin) [reply tools/sigh.ts]: Similar done.

**Pull request merged**